### PR TITLE
Moving translations and templates folders to root folder

### PR DESCRIPTION
**Pull request by mstrom**
Symfony 4 will have templates folder under root folder
So I'm proposing to move templates to root folder for simplicity and translations as well

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ framework:
     # and with different cache configurations for each fragment
     # https://symfony.com/doc/current/book/http_cache.html#edge-side-includes
     esi:             { enabled: true }
-    translator:      { fallback: '%locale%' }
+    translator:      { fallback: '%locale%', paths: ["%kernel.project_dir%/translations"] }
```
Comment (yceruto) [inline app/config/config.yml]: Single quotes

File: app/config/config.yml [hunk 2]
```diff
@@ -64,6 +64,9 @@ twig:
     form_themes:
         - 'form/layout.html.twig'
         - 'form/fields.html.twig'
+    paths:
+        "%kernel.project_dir%/templates/TwigBundle": Twig
+        "%kernel.project_dir%/templates": ~
```
Comment (yceruto) [inline app/config/config.yml]: Same here  ;)

Comment (mstrom) [inline app/config/config.yml]: fixed

File: app/config/config.yml (hunk 1 above)
Comment (mstrom) [inline app/config/config.yml]: fixed

**Comment by javiereguiluz** [comment]
@mstrom thanks A LOT for this contribution. Sadly, I must close for now as "won't fix" for the reasons explained by @yceruto. This is the dir structure used by Flex/Symfony 4 ... but we are not using any of those yet (we can't until November 2017).

A similar, but different, proposal is #602 which moves assets dir. This is because the decision is based on Webpack Encore, which is a library independent from Flex/Sf4 and which we already use in this app.

Thanks for understanding it!

**Pull request closed without merging**